.travis.yml

```diff
@@ -42,11 +43,11 @@ jobs:
         - molo_version="$(sed -nE 's/^molo\.core==(.*)$/\1/p' requirements.txt)"
         - docker pull "$IMAGE_NAME" || true
```

Comment:
I don't think there's actually much that can be cached between Docker images so I think it's right to remove the `--cache-from` below. You can also remove this line which pulls the image to be used for `--cache-from`.